### PR TITLE
systemd: 247.6 -> 248.3

### PR DESCRIPTION
**Pull request by Mic92**
###### Motivation for this change

This should be not merged before the branch off the next release.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Mic92** [comment]
I had some tty issues still. Need to fix those on a different machine...

File: pkgs/os-specific/linux/systemd/default.nix [hunk 1]
```diff
@@ -221,6 +216,16 @@ stdenv.mkDerivation {
 
         # journalctl --grep requires libpcre so lets provide it
         { name = "libpcre2-8.so.0"; pkg = pcre2; }
+
+        # TODO: Support for TPM2 in systemd-cryptsetup, systemd-repart and systemd-cryptenroll
```
Comment (Mic92) [inline pkgs/os-specific/linux/systemd/default.nix:220]: Also need to look into this...

File: pkgs/os-specific/linux/systemd/default.nix [hunk 2]
```diff
@@ -151,12 +152,6 @@ stdenv.mkDerivation {
     ./0017-path-util.h-add-placeholder-for-DEFAULT_PATH_NORMAL.patch
     ./0018-logind-seat-debus-show-CanMultiSession-again.patch
     ./0019-pkg-config-derive-prefix-from-prefix.patch
-
-    # Fix -Werror=format.
-    (fetchpatch {
```
Comment (SuperSandro2000) [inline pkgs/os-specific/linux/systemd/default.nix:156]: Please remove the unused fetchpatch from inputs.

**Comment by dotlambda** [comment]
It would be nice to see the remaining issues solved and the PR merged soon since this update fixes https://github.com/systemd/systemd/issues/19604.

**Comment by globin** [comment]
Any update on this? What are the current issues?

**Comment by SebTM** [comment]
I'm also waiting for this because of the introduced feature "ActivationPolicy" - so I can manually manage when/which wireguard-connection should be up without rebuild. Please note that there is also another pull-request for system-upgrade (https://github.com/NixOS/nixpkgs/pull/119790)

**Comment by tijolao** [comment]
I think they are waiting 249 to be release, that I think is very close to happen.

**Comment by SebTM** [comment]
Thx for the feedback 👍🏻 :)

**Comment by flokli** [comment]
I think @Mic92 referred to the nixos branchoff, which has already happened.

@Mic92 do you want to pick this up? I would prefer to stabilize 248 before jumping to 249, just haven't had enough of time recently :-)

**Comment by Mic92** [comment]
I am closing this for now to encourage someone else to pick it up. I am not sure when I would get to this. Feel free to pick up my process.

**Pull request closed without merging**